pywbem_mock/_mainprovider.py

```diff
@@ -502,11 +502,18 @@ def _get_instance(self, namespace, instance_name, instance_store,
         Parameters:
 
           namespace (:term:`string`):
-              Namespace containing the instance. This is used only for
-              explanatory data in exception explanations
+            Namespace containing the instance. This is used only for
+            explanatory data in exception explanations
```

Comment:
Using the namespace parameter for diag purposes when the instance_name parameter is required to have the namespace set which is used for functional purposes does not seem right.

I suggest to remove the namespace parameter also from this internal method and use it from the instance_name parameter:
```
namespace = instance_name.namespace
```

Reply:
Sorry, I did not see that this was already done in the next commit. Closed.